upi/metal/main.tf

```
@@ -1,12 +1,25 @@
 # ================IGNITION=====================
 
-# force local ignition provider binary
-provider "ignition" {
-  version = "0.0.0"
+terraform {
+  required_providers {
+    ignition = {
+      source  = "community-terraform-providers/ignition"
+      version = "2.1.2"
+    }
+  }
```

We can have all of the required providers in one block, so something like:

```suggestion
  required_providers {
    ignition = {
      source  = "community-terraform-providers/ignition"
      version = "2.1.2"
    }
    
    metal = {
      source = "equinix/metal"
      version = "3.2.2"
    }
    
    matchbox = {
         source  = "poseidon/matchbox"
      version = "0.5.0"
    }
  }
```